imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
+    error, status, movie_list = SearchResults.get_search_movies(movie)
```

Comment:
in python we can use this syntax .. and here we have used three variables in a row as SearchResults.get_search_movies(movie) returns three values (a tuple having three elements).. and  these values are here assigned in a given order .. the error (true/False).. status - (503/404/200) movies_list which is a  json object data